wdl/ExpansionHunter.wdl

```diff
@@ -278,3 +339,67 @@ task ConcatEHOutputs {
         maxRetries: select_first([runtime_attr.max_retries, runtime_default.max_retries])
     }
 }
+
+task RunReviewer {
+    input {
+        String sample_id
+        File realigned_bam
+        File realigned_bam_index
+        File vcf_gz
+        File reference_fasta
+        File? reference_fasta_index
+        File variant_catalog_json
+        String expansion_hunter_docker
+        RuntimeAttr? runtime_override
+    }
+
+    output {
+        File? reviewer_images_gz = "reviewer_images.tar.gz"
+        File? sample_metrics = "${sample_id}_metrics.csv"
+    }
+
+    command <<<
+        set -euxo pipefail
+
+        REF="$(basename "~{reference_fasta}")"
+        mv ~{reference_fasta} $REF
+        mv ~{reference_fasta_index} $REF.fai
```

Comment:
Usually we try to soft-link files rather than use `mv` in case the user is on a shared file system.

Reply:
That is a better alternative, thanks!